mlir/include/mlir/Transforms/Passes.h

```diff
@@ -130,6 +130,13 @@ createSymbolPrivatizePass(ArrayRef<std::string> excludeSymbols = {});
 /// their producers.
 std::unique_ptr<Pass> createTopologicalSortPass();
 
+/// Create composite pass, which runs selected set of passes until fixed point
```

Comment:
Nit: I would include "fixed point" in the name, "composite" is very generic.

Reply:
Renamed to `CompositeFixedPointPass`, suggestions for better naming are welcome :)